### ISSUE TITLE
Formato entrega T02

### ISSUE THREAD

**Issue by fdominguezclaro** [IIC2343/syllabus#44]
Hola!

Aparte del orden de inputs y outputs que están especificados en el enunciado, es necesario seguir algún formato estricto (nombre del script por ejemplo)? Puedo editar un poco el inicio del script que subieron? Como por ejemplo imprimir el nombre de lo que se pide al escribir un input.

**Comment by FranLucchini** [comment]
No, deben seguir el mismo formato al momento de entregar. Por mientras, pueden imprimir otras cosas, pero después debe quedar igual al formato dado en el enunciado.
